### ISSUE TITLE
Throats from face pore body to face pore body for FaceCenteredCubic

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#2772]
### Discussed in https://github.com/PMEAL/OpenPNM/discussions/2770

<div type='discussions-op-text'>

<sup>Originally posted by **AnnaMareike** June 19, 2023</sup>
Hi all,

If I use `op.network.FaceCenteredCubic(...)` to create a network, throats which are connected to two pore bodies at a face are currently labeled with `corner_to_face`.

Shouldn't they be labeled as `face_to_face`? (For `BodyCenteredCubic` respective throats also get labeled `body_to_body`.)

I'm just wondering if this is a bug, which needs to be fixed or if this is on purpose and I should relabel the respective throats if I want them to be labeled as `face_to_face`.

**Plots for understanding**
This is the plot created for the `FaceCenteredCubic` , where also `corner_to_face` (red) throats can be found between two `face` (green)pore bodies. This seems wrong in my opinion.
![Figure_fcc](https://github.com/PMEAL/OpenPNM/assets/74598829/f02b1284-cae0-4f93-9589-a7b3886e821b)

`BodyCenteredCubic` : Here `body_to_body` (green) throats can be correctly found between two `body` (green) pore bodies
![Figure_bcc](https://github.com/PMEAL/OpenPNM/assets/74598829/c7c05e19-5232-496f-9cd0-5996fe159f62)

**Code to reproduce plots**
```Python
import openpnm as op
import matplotlib.pyplot as plt

network_fcc = op.network.FaceCenteredCubic(shape=[3, 2, 2])
network_bcc = op.network.BodyCenteredCubic(shape=[3, 2, 2])

pores_corner_fcc = network_fcc['pore.corner']
pores_face_fcc = network_fcc['pore.face']
throats_corner_to_face_fcc = network_fcc['throat.corner_to_face']
throats_corner_to_corner_fcc = network_fcc['throat.corner_to_corner']

pores_corner_bcc = network_bcc['pore.corner']
pores_body_bcc = network_bcc['pore.body']
throats_corner_to_body_bcc = network_bcc['throat.corner_to_body']
throats_corner_to_corner_bcc = network_bcc['throat.corner_to_corner']
throats_body_to_body_bcc = network_bcc['throat.body_to_body']


plot_network = True

if plot_network:
    ax1 = op.visualization.plot_connections(network_fcc, throats=throats_corner_to_face_fcc, color = 'r', label = 'corner_to_face throat')
    ax1 = op.visualization.plot_connections(network_fcc, ax=ax1, throats=throats_corner_to_corner_fcc, color = 'b' , label = 'corner_to_corner throat')
    ax1 = op.visualization.plot_coordinates(network_fcc, ax=ax1, pores=pores_corner_fcc, c='b', label = 'corner pore body')
    ax1 = op.visualization.plot_coordinates(network_fcc, ax=ax1, pores=pores_face_fcc, c='g', label = 'face pore body')
    plt.legend()
    plt.show()

    ax2 = op.visualization.plot_connections(network_bcc, throats=throats_corner_to_body_bcc , color = 'r', label = 'corner_to_body throat')
    ax2 = op.visualization.plot_connections(network_bcc, ax=ax2, throats=throats_corner_to_corner_bcc, color = 'b' , label = 'corner_to_corner throat')
    ax2 = op.visualization.plot_connections(network_bcc, ax=ax2, throats=throats_body_to_body_bcc, color = 'g' , label = 'body_to_body throat')
    ax2 = op.visualization.plot_coordinates(network_bcc, ax=ax2, pores=pores_corner_bcc, c='b', label = 'corner pore body')
    ax2 = op.visualization.plot_coordinates(network_bcc, ax=ax2, pores=pores_body_bcc, c='g', label = 'body pore body')
    plt.legend()
    plt.show()
```

</div>

**Comment by jgostick** [comment]
Ok, I have fixed it and will push/merge soon.  The following code gives the images below

``` python

import openpnm as op
import matplotlib.pyplot as plt

fcc = op.network.FaceCenteredCubic(shape=[3, 2, 2])
bcc = op.network.BodyCenteredCubic(shape=[3, 2, 2])

ax1 = op.visualization.plot_connections(network=fcc, throats=fcc['throat.corner_to_face'], color='r', label='corner_to_face throat')
ax1 = op.visualization.plot_connections(network=fcc, ax=ax1, throats=fcc['throat.corner_to_corner'], color='b' , label='corner_to_corner throat')
ax1 = op.visualization.plot_connections(network=fcc, ax=ax1, throats=fcc['throat.face_to_face'], color='g' , label='face_to_face throat')
ax1 = op.visualization.plot_coordinates(network=fcc, ax=ax1, pores=fcc['pore.corner'], c='b', label='corner pore body')
ax1 = op.visualization.plot_coordinates(network=fcc, ax=ax1, pores=fcc['pore.face'], c='g', label='face pore body')
plt.legend()
plt.show()

ax2 = op.visualization.plot_connections(network=bcc, throats=bcc['throat.corner_to_body'] , color='r', label='corner_to_body throat')
ax2 = op.visualization.plot_connections(network=bcc, ax=ax2, throats=bcc['throat.corner_to_corner'], color='b' , label='corner_to_corner throat')
ax2 = op.visualization.plot_connections(network=bcc, ax=ax2, throats=bcc['throat.body_to_body'], color='g' , label='body_to_body throat')
ax2 = op.visualization.plot_coordinates(network=bcc, ax=ax2, pores=bcc['pore.corner'], c='b', label='corner pore body')
ax2 = op.visualization.plot_coordinates(network=bcc, ax=ax2, pores=bcc['pore.body'], c='g', label='body pore body')
plt.legend()
plt.show()

```
![bcc](https://github.com/PMEAL/OpenPNM/assets/5092049/b33ec228-89c7-4583-a2e9-e741da8228e5)


![fcc](https://github.com/PMEAL/OpenPNM/assets/5092049/75da0594-f139-4b72-9eb5-f86426866796)

**Comment by jgostick** [comment]
This was fixed on PR #2774
